app/src/main/java/de/ur/explure/viewmodel/AuthenticationViewModel.kt

```diff
@@ -15,20 +14,20 @@ class AuthenticationViewModel(
     private val mainAppRouter: MainAppRouter
 ) : ViewModel() {
 
-    private val _toast = MutableLiveData<String?>()
-    val toast: LiveData<String?> = _toast
+    private val mutableToast = MutableLiveData<String?>()
+    val toast: LiveData<String?> = mutableToast
 
     fun signIn(email: String, password: String) {
         viewModelScope.launch {
             when (val loginTask = authService.signIn(email, password)) {
                 is FirebaseResult.Success -> {
-                    // do task
+                    // doesn't need to do anything
                 }
                 is FirebaseResult.Error -> {
-                    _toast.value = loginTask.exception.message
+                    mutableToast.value = loginTask.exception.message
                 }
                 is FirebaseResult.Canceled -> {
-                    _toast.value = loginTask.exception!!.message
+                    mutableToast.value = loginTask.exception!!.message
```

Comment:
Prinzipiell ist !! in Kotlin (fast) immer schlechter Stil und sollte dementsprechend (fast) nie verwendet werden. Hier könntest du stattdessen auch einfach ein ? schreiben. Musst du jetzt nicht extra überall ändern, nur ein Hinweis.